ansible/setup.yml

```diff
@@ -6,14 +6,20 @@
   - name: find the ip of docker-machine
     local_action: shell "docker-machine" "ip" "{{docker_machine_name | default('whisk')}}"
     register: result
-    ignore_errors: true
+    when: "'environments/mac' in inventory_dir"
```

Comment:
I'ld rather not hardcode an environment here. We had that in ant before, problems and strange behaviour inevitable. Couldn't we just check if `docker-machine` is available as a command?